src/foam/nanos/controller/ApplicationController.js

```diff
@@ -832,44 +834,21 @@ foam.CLASS({
     async function checkGeneralCapability() {
       var groupDAO = this.__subContext__.groupDAO;
       var group = await groupDAO.find(this.subject.user.group);
-      if ( group && group.generalCapability != '' ) {
-        const ucjCheck = async () => await this.__subContext__.crunchService.getJunction(null, group.generalCapability);
-        var ucj = await ucjCheck();
-
-        if ( ucj == null || ucj.status != this.CapabilityJunctionStatus.GRANTED ) {
-
-          const lastWizard = this.crunchController.lastActiveWizard;
-          if ( lastWizard &&
-            lastWizard.status === this.crunchController.WizardStatus.IN_PROGRESS
-          ) {
-            let x = this.__subContext__.createSubContext({
-              wizardController: lastWizard
-            });
-            const seq = await this.crunchController.createUCJInlineWizardSequence(x)
-              .addBefore('CapabilityAdaptAgent', { class: 'foam.u2.wizard.agents.RootCapabilityAgent', rootCapability: group.generalCapability})
-              ;
-
-            lastWizard.status$.sub(async () => {
-              if ( lastWizard.status !== this.crunchController.WizardStatus.IN_PROGRESS ) {
-                await this.doGeneralCapabilityPostCheck(ucjCheck);
-              }
-            })
-
-            await this.crunchController.inlineWizardFromSequence(lastWizard, seq);
-
-          } else {
-            await this.crunchController.createTransientWizardSequence(this.__subContext__)
-              .addBefore('ConfigureFlowAgent', { class: 'foam.u2.wizard.agents.RootCapabilityAgent', rootCapability: group.generalCapability})
-              .reconfigure('WAOSettingAgent', { waoSetting: foam.u2.crunch.wizardflow.WAOSettingAgent.WAOSetting.UCJ })
-              .remove('RequirementsPreviewAgent')
-              .execute();
-            
-            await this.doGeneralCapabilityPostCheck(ucjCheck);
-          }
 
-        }
+      if ( ! group || ! group.generalCapability ) return true;
```

Comment:
```suggestion
      if ( ! group?.generalCapability || ! group?.wizardFlow ) return true;
```

Reply:
we actually need `generalCapability` to be set as well so we know which granted status to check for